parsons/etl/tofrom.py

```diff
@@ -98,6 +98,142 @@ def to_html(
 
         return local_path
 
+    def to_avro(
+        self, target, schema=None, sample=9, codec="deflate", compression_level=None, **avro_args
+    ):
+        """
+        Outputs table to an Avro file.
+
+        In order to use this method, you must have the `fastavro` library installed.
+        If using limited dependencies, you can install it with `pip install parsons[avro]`.
+
+        Write the table into a new avro file according to schema passed.
+
+        This method assume that each column has values with the same type
+        for all rows of the source `table`.
+
+        `Apache Avro`_ is a data
```

Comment:
I see a lot of links syntax with no corresponding links - can you add them? (Or are you using RST in a way I'm unfamiliar with?) 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links